Header Header
float64 x
float64 y
float64 vx
float64 vy
cartbot/Cluster[] objects
uint8 state
int32 lost_cnt
uint8 STOP=0
uint8 LOST=1
uint8 COUNT=2
uint8 INIT=3
uint8 TRACKING=4